<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneListPreviewDialog</class>
 <widget class="QDialog" name="SceneListPreviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
   <item>
    <layout class="QHBoxLayout" name="framePreviewsHorizontalLayout" stretch="2,3,3,2">
     <item>
      <layout class="QVBoxLayout" name="firstSceneStartPreviewVerticalLayout" stretch="1,2,0">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="OpenCVImageWidget" name="firstSceneStartPreviewOpenCVImageWidget" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="firstSceneStartPreviewCurrentFrameNumberLabel">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="firstSceneEndPreviewVerticalLayout" stretch="1,0">
       <item>
        <widget class="OpenCVImageWidget" name="firstSceneEndPreviewOpenCVImageWidget" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="firstSceneEndPreviewCurrentFrameNumberLabel">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="secondSceneStartPreviewVerticalLayout" stretch="1,0">
       <item>
        <widget class="OpenCVImageWidget" name="secondSceneStartPreviewOpenCVImageWidget" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="secondSceneStartPreviewCurrentFrameNumberLabel">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="secondSceneEndPreviewVerticalLayout" stretch="1,2,0">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="OpenCVImageWidget" name="secondSceneEndPreviewOpenCVImageWidget" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="secondSceneEndPreviewCurrentFrameNumberLabel">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="currentSceneControlsHorizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="currentSceneControlsGoStartButton">
       <property name="text">
        <string>|&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="currentSceneControlsStepBackButton">
       <property name="text">
        <string>&lt;|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentSceneControlsLabel">
       <property name="text">
        <string>0 / 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="currentSceneControlsStepForwardButton">
       <property name="text">
        <string>|&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="currentSceneControlsGoEndButton">
       <property name="text">
        <string>&gt;|</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dialogButtonsHorizontalLayout">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="dialogCloseButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenCVImageWidget</class>
   <extends>QWidget</extends>
   <header>opencv_image_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dialogCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>SceneListPreviewDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
